nvi-commons/src/testFixtures/java/no/sikt/nva/nvi/common/model/NviCreatorFixtures.java

```diff
@@ -0,0 +1,35 @@
+package no.sikt.nva.nvi.common.model;
+
+import static no.sikt.nva.nvi.common.model.ContributorFixtures.STATUS_UNVERIFIED;
+import static no.sikt.nva.nvi.common.model.ContributorFixtures.STATUS_VERIFIED;
+import static no.sikt.nva.nvi.test.TestUtils.randomUriWithSuffix;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+
+import java.util.List;
+import java.util.stream.Stream;
+import no.sikt.nva.nvi.common.client.model.Organization;
+import no.sikt.nva.nvi.common.db.CandidateDao.DbCreatorType;
+import no.sikt.nva.nvi.common.service.dto.UnverifiedNviCreatorDto;
+import no.sikt.nva.nvi.common.service.dto.VerifiedNviCreatorDto;
+
+public class NviCreatorFixtures {
+
+  public static List<DbCreatorType> mapToDbCreators(
+      List<VerifiedNviCreatorDto> verifiedNviCreators,
+      List<UnverifiedNviCreatorDto> unverifiedNviCreators) {
+    Stream<DbCreatorType> verifiedCreators =
+        verifiedNviCreators.stream().map(VerifiedNviCreatorDto::toDao);
+    Stream<DbCreatorType> unverifiedCreators =
```

Comment:
var, var

Reply:
Well yes, but:

```
Required type:
List
<DbCreatorType>
Provided:
List
<? extends Record>
```

I'm probably doing something stupid here, but Java is struggling to understand that our records are implementing the same interface. We have the same issue in many places.